go.mod

```diff
@@ -27,7 +27,10 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
-require github.com/onsi/ginkgo/v2 v2.7.0
+require (
+	github.com/IBM/vpc-go-sdk v0.35.0
+	github.com/onsi/ginkgo/v2 v2.8.1
+)
```

Comment:
Any specific reason for creating a different block? if not - can this be merged with above block?